src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/iso.mdx

```diff
@@ -60,6 +60,10 @@ The following applies to the New Relic platform:
   </tbody>
 </table>
 
+<Callout variant="important">
+If you require a copy of New Relic's full ISO 27001 certification letter, reach out to your New Relic account executive.
```

Comment:
Shouldn't it be "account manager" or "account representative"?

Reply:
I'm seeing "account representative" used most often in our docs

Reply:
That's what I thought as well! I co-wrote this with someone in legal and apparently account executive is the correct term to use.